docs/tools/getting-started.md

```diff
@@ -4,21 +4,29 @@ title: Getting Started
+Here you will find tutorials and tools that help you build on LUKSO. As LUKSO is an EVM-based Blockchain, all tools and tutorials for Ethereum also work well for LUKSO. If you need EVM and Smart Contract knowledge, we advise you to look at [these great resources from the Ethereum Foundation](https://ethereum.org/en/developers/learning-tools/).
 
-Other than most EVM chains, you're encouraged on LUKSO to mainly use [Universal Profiles](../standards/universal-profile/introduction.md) as the main account of your dApps and as a gateway for your users to the LUKSO Blockchain. It is discouraged to use simple EOAs as accounts, as they are insecure, inflexible and don't track incoming assets.
+Other than most EVM chains, you're encouraged on LUKSO to mainly use [Universal Profiles](../standards/universal-profile/introduction.md) as the account solution for dApps and as a gateway for your users to the LUKSO Blockchain. It is discouraged to use simple EOAs as accounts, as they are insecure, inflexible and don't track incoming assets. If you need a more general understanding of why we build our accounts on the smart contract level, we advise you to read about the [LUKSO Ecosystem](https://medium.com/lukso/lukso-ecosystem-part-1-4c3f5d67b081) concept.
 
 ## UP in 1-2-3
 
 This little tutorial will show you how to deploy and interact with a UniversalProfile.
-You can:
+To maximize your learning, you can:
 
-- run this tutorial in the console of <https://universalprofile.cloud>, which has all necessary contract objects available (under the `contracts` object)
-- clone the [`up-sample-react-app`](https://github.com/Hugoo/up-sample-react-app) repo and work in it
-- write your own JS/TS scripts
-- check the [demo app](https://hugoo.github.io/up-sample-react-app/)
+- run this tutorial in the console of the [profile explorer](https://universalprofile.cloud),
+- clone the [`up-sample-react-app`](https://github.com/Hugoo/up-sample-react-app) repo and work in it,
+- write your own JavaScript or TypeScript code snippets on top, and
+- check the [demo app](https://hugoo.github.io/up-sample-react-app/) for some first impressions.
 
-First, generate a key that will control your UP and fund it via the [L14 Faucet](http://faucet.l14.lukso.network):
+:::info
+The [profile explorer](https://universalprofile.cloud) has all necessary entities under the `contracts` object. Take a look at different profiles and assets you want to experiment with within your code.
+:::
+
+:::note
+We will use the [`@erc725/erc725.js`](./erc725js/getting-started) NPM package in order to get quick accsess to reading keys, values and ABIs.
+:::
+
+#### Generate a key that will control your Universal Profile.
```

Comment:
Why go from `##` to `####` and skip `###` ?

Reply:
We do this in other guides as well to declare a sentence as a sub-step that won't mess up the menu bar on the right side

before:
![before](https://user-images.githubusercontent.com/61689369/165263337-07fe4115-cdea-4ce1-9380-23e504333905.png)

after:
![after](https://user-images.githubusercontent.com/61689369/165263001-cbf4027b-4df8-48d3-a46a-811845bfbd59.png)

Reply:
ok thks